dev/release-notes.py

```diff
@@ -0,0 +1,211 @@
+            f.write(issuenumber)
+            f.write("](")
+            f.write("https://github.com/gap-system/gap/pull/" + str(k))
+            f.write(") ")
+            f.write(prs[k]["title"])
+            f.write("\n")
+            removelist.append(k)
+    for item in removelist:
+        del prs[item]
+    f.write("\n")
+
+
+    removelist = []
+    for k in prs:
+        if "release notes: not needed" in prs[k]["labels"]:
+            removelist.append(k)
+    for item in removelist:
+        del prs[item]
+        del jsondict[item]
+
+
+    f2.write("### " + "release notes: to be added" + "\n")
+    removelist = []
+    for k in prs:
+        if "release notes: to be added" in prs[k]["labels"]:
+            f2.write("- [#")
+            issuenumber = str(k)
+            f2.write(issuenumber)
+            f2.write("](")
+            f2.write("https://github.com/gap-system/gap/pull/" + str(k))
+            f2.write(") ")
+            f2.write(prs[k]["title"])
+            f2.write("\n")
+            removelist.append(k)
+    for item in removelist:
+        del prs[item]
+    f2.write("\n")
+
+
+    f2.write("### Uncategorized PR" + "\n")
+    removelist = []
+    for k in prs:
+        #if not "release notes: use title" in item[2]:
+        if not "release notes: added" in prs[k]["labels"]:
+            f2.write("- [#")
+            issuenumber = str(k)
+            f2.write(issuenumber)
+            f2.write("](")
+            f2.write("https://github.com/gap-system/gap/pull/" + str(k))
+            f2.write(") ")
+            f2.write(prs[k]["title"])
+            f2.write("\n")
+            removelist.append(k)
+    for item in removelist:
+        del prs[item]
+    f2.close()
+    
+
+    for priorityobject in prioritylist:
+        f.write("### " + priorityobject[1] + "\n")
+        removelist = []
+        for k in prs:
+            if priorityobject[0] in prs[k]["labels"]:
+                f.write("- [#")
+                issuenumber = str(k)
+                f.write(issuenumber)
+                f.write("](")
+                f.write("https://github.com/gap-system/gap/pull/" + str(k))
+                f.write(") ")
+                f.write(prs[k]["title"])
+                f.write("\n")
+                removelist.append(k)
+        for item in removelist:
+            del prs[item]
+        f.write("\n")
+    f.close()
+
+    f3.write("[")
+    jsonlist = []
+    for k in jsondict:
+        temp = []
+        temp.append(str(jsondict[k]["title"]))
+        temp.append(str(k))
+        temp.append(jsondict[k]["labels"])
+        jsonlist.append(temp)
+    for item in jsonlist:
+        f3.write("%s\n" % item)
+    f3.write("]")
+    f3.close
+
+def main(startdate):
+
+    # Authentication and checking current API capacity
+    # TODO: for now this will do, use Sergio's code later
+    with open("/Users/alexk/.github_shell_token", "r") as f:
+        accessToken=f.read().replace("\n", "")
+    g=Github(accessToken)
```

Comment:
One last comment for anybody using this script: 
You have two options to connect with Github: 
1. Use your username and password 
2. Use an access token (can be generated in settings -> Developer Settings -> Personal access token)
In any case, use the second variant with the token. If you otherwise push your version of the script, the password is in plaintext on Github.